recipes/openms/meta.yaml

```diff
@@ -1,21 +1,26 @@
 {% set name = "OpenMS" %}
-{% set version = "2.4.0" %}
+{% set version = "2.5.0" %}
 package:
   name: {{ name|lower  }}
   version: {{ version  }}
 
 source:
-  url: https://github.com/OpenMS/OpenMS/releases/download/Release{{ version }}/OpenMS-{{ version }}-pyopenmsfixes.tar.gz
-  md5: 79445708ac44bdbae5ef95322e4b39b2
+  url: https://abibuilder.informatik.uni-tuebingen.de/archive/openms/OpenMSInstaller/RC/2.5.0/OpenMS-2.5.0-src-conda_test.tar.gz
+  md5: 9d8d50c0d9d41e993a602607436840ae
```

Comment:
Maybe switch to sha256

Reply:
Yes, will do when we link to the GitHub Release.